src/components/view/Navigation/index.js

```diff
@@ -58,23 +44,40 @@ class Navigation extends React.Component {
         this.props.history.push(tabEl.props['data-route']);
     };
 
+    getMenuItemHandler (routeName)
+    {
+        var that = this;
```

Comment:
Why use `var that = this;`? There should be a more `reactjs` way of doing this using either fat arrow functions or setting context with `.bind()` in the constructor. See other examples across app.